confobo/controllers/subscriptions.py

```diff
@@ -2,7 +2,7 @@
 from confobo.persistence import subscriptions
 
 
-def remove_all(user: User) -> bool:
+def unsubscribe_user(user: User) -> bool:
```

Comment:
Yeah, verbose, probably. Actually, I just want it to have `unsubscribe` in the name, because at some point I imported `remove_all`, couldn't figure out what it removes and got confused. Apart from using the word `unsubscribe`, I don't care.